setup.py

```diff
@@ -33,10 +33,10 @@
 ]
 
 if PY3:
-    install_requires.append("Twisted >= 15.5.0")
+    install_requires.append("Twisted >= 16.0.0")
```

Comment:
These versions no longer differ, and thus should be moved out of the conditional and into the statically declared `install_requires`.